tests/e2e/utils/mock-requests.js

```diff
@@ -559,6 +639,7 @@ export default class MockRequests {
 			status: 'connected',
 			notification_service_enabled: notificationServiceEnabled,
 			wpcom_rest_api_status: wpcomRestApiStatus,
+			step: '',
```

Comment:
The `''` step doesn't seem to exist in the Google Merchant Center connection API?